Userland/Libraries/LibELF/DynamicLoader.h

```diff
@@ -133,7 +137,8 @@ class DynamicLoader : public RefCounted<DynamicLoader> {
         Success = 1,
         ResolveLater = 2,
     };
-    RelocationResult do_relocation(DynamicObject::Relocation const&, ShouldInitializeWeak should_initialize_weak);
+    RelocationResult do_direct_relocation(DynamicObject::Relocation const&, ShouldInitializeWeak);
+    static RelocationResult do_plt_relocation(DynamicObject::Relocation const&);
```

Comment:
This comment was lost, linking this to the fixup function. Perhaps the friend declaration should be closeby as well?

```
// Will be called from _fixup_plt_entry, as part of the PLT trampoline
```